tests/models/test_svc.py

```diff
@@ -0,0 +1,121 @@
+from numpy.testing import assert_allclose, assert_equal
+from sklearn.svm import SVC as SkSVC
+from scratchml.models.svc import SVC
+from scratchml.scalers import StandardScaler
+from ..utils import generate_classification_dataset, repeat
+import unittest
+import numpy as np
+import math
+
+
+class Test_SVC(unittest.TestCase):
+    """
+    Unit test class for the custom SVC implementation.
+    """
+
+    @repeat(3)
+    def test_binary_classification(self):
```

Comment:
It would be nice to add a few more validations, such as asserting the output type and shape of both implementations are equal, other parameters value, and so on. Check out other implementation tests to see if you get any idea to increment this.